contribute/architecture/frontend-data-requests.md

```diff
@@ -0,0 +1,51 @@
+# Data requests
+
+[BackendSrv is the core data request service](https://grafana.com/docs/grafana/latest/packages_api/runtime/backendsrv) used by Grafana. This documentation highlights the high level concepts used by `BackendSrv`.
+
+## Request `cancellation`
```

Comment:
```suggestion
## Canceling requests
```

It's not obvious to me what this section is about, especially with the backticks around cancellation.

Cancelling with two 'L's is British spelling. We prefer American spelling in our docs. Please update this throughout the document.

Reply:
Sure, will do!